tensorrt_llm/_torch/modules/fused_moe/quantization.py

```
@@ -2079,10 +2112,11 @@ def get_weights_shapes(self, module: torch.nn.Module, weight_vec_size: int,
 
     def create_weights(self, module: torch.nn.Module):
         weight_vec_size = torch.iinfo(self.weight_dtype).bits // 4
+        self.block_scales_vec_size = torch.iinfo(self.block_scales_dtype).bits // 8
         block_scales_vec_size = torch.iinfo(self.block_scales_dtype).bits // 8
```

Ruff (0.14.8)

Local variable `block_scales_vec_size` is assigned to but never used

Remove assignment to unused variable `block_scales_vec_size`

(F841)